docs/schema-update-guide.md

```diff
@@ -1,23 +1,38 @@
 # Schema Update Guide
 
-All schemas are located at [store/migration](https://github.com/bytebase/bytebase/tree/main/store/migration). Bytebase schema release follows semantic versioning. This document goes over the scheme update process during development and release.
+This document goes over the scheme update process during development and production release.
 
-We use different schemas for [dev](https://github.com/bytebase/bytebase/tree/main/store/migration/dev) and [release](https://github.com/bytebase/bytebase/tree/main/store/migration/release). The `LATEST.sql` and `LATEST_DATA.sql` will be the schema and data to initialize a new database. The version directory such as `1.1` contains all the DDL statements used to migrate from previous version of schema such as `1.0`. There should only one version in dev schema directory.
+All schemas are located at [store/migration](https://github.com/bytebase/bytebase/tree/main/store/migration). Bytebase schema release follows semantic versioning.
 
-## Development
+Prod
+
+- [prod](https://github.com/bytebase/bytebase/tree/main/store/migration/prod) schema.
+- `LATEST.sql` and `LATEST_DATA.sql` will be the schema and data to initialize a new database. 
+- The version directory such as `1.1` contains all the DDL statements used to migrate from previous version of schema such as `1.0`. When a prod release binary starts, it will apply all the schema migrations from current version up to the latest schema release version.
+
+Dev
+
+- [dev](https://github.com/bytebase/bytebase/tree/main/store/migration/dev) schema.
+- `LATEST.sql` and `LATEST_DATA.sql` will only be used to view the latest schema.
+- When a dev release binary starts, it will use the LATEST.sql in release directory to initialize the database and then apply all the DDLs in the dev directory if not yet.
```

Comment:
```suggestion
- When a dev release binary starts, it will use the LATEST.sql in the release directory to initialize the database and then apply all the DDLs in the dev directory if not yet applied.
```